deploy/charts/cert-manager/templates/deployment.yaml

```diff
@@ -154,14 +154,18 @@ spec:
           {{- if or .Values.config .Values.volumeMounts }}
           volumeMounts:
             {{- if .Values.config }}
-            - name: config 
+            - name: config
               mountPath: /var/cert-manager/config
             {{- end }}
             {{- with .Values.volumeMounts }}
             {{- toYaml . | nindent 12 }}
             {{- end }}
           {{- end }}
           env:
+          # TODO: We can remove this when we upgrade to client-go@v0.31.0
+          # See https://github.com/kubernetes/kubernetes/commit/9248cccc27fdd52a9a99fd9ad6e47ada9ee8b568
+          - name: ENABLE_CLIENT_GO_WATCH_LIST_ALPHA
+            value: "1"
```

Comment:
* https://github.com/kubernetes/kubernetes/pull/122791

Reply:
I removed this section, since we are now using `client-go@v0.31.0`.